xarray/core/options.py

```diff
@@ -91,26 +94,35 @@ def _positive_integer(value: int) -> bool:
     return isinstance(value, int) and value > 0
 
 
+def _is_boolean(value: Any) -> bool:
+    return isinstance(value, bool)
```

Comment:
minor comment: This is a matter of taste but this feels like a fairly tiny benefit for the added indirection. The diff would be a lot smaller too, if you just followed the existing copy-paste approach.

Reply:
I tried to apply "opportunistic refactoring", since I was updating this zone of the code. Currently, there is a mixed approach: lambdas are used, but a function `_positive_integer` already exists. So I tried to align all the existing checks to `_positive_integer`. It looks at least to me a little bit cleaner to refer multiple times one function rather than define multiple duplicated lambdas.
I understand however the main downside of it adding unrelated noise to the diff, sorry for that. But now that the change is made, should I revert it or keep it?